### DISCUSSION TITLE
Add Find in Page to Code

### DISCUSSION THREAD

**Discussion by gmt2001** [community/community#4134] [Mobile] [answered]
When viewing code, it is often helpful to to have the "Find in Page" feature from the browser to find a particular function or section of code.

One use case is when looking to quickly answer a question or comment that was asked in a Discord server. A quick look at that section of code, when the general file and keywords are known, in the context of the question or comment can help provide the appropriate answer, or reveal a mistake that warrants opening a ticket.

It is therefore requested to add the "Find in Page" feature to the GitHub for Android/iOS apps when viewing the source of a code file

**Comment by aconsuegra** [comment, marked as answer]
Thank you so much for the request! We are tracking being able to add search capabilities to our code view internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it!

**Reply by oliveratgithub** [reply]
+1 for having a "Find" feature in code files within the GitHub Mobile App

**Reply by thiagohmcruz** [reply]
+1 to this!

**Reply by IWareQ** [reply]
+1 to this

**Comment by iamjonny** [comment]
> Thank you so much for the request! We are tracking being able to add search capabilities to our code view internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it!

Hi, do you know if this is planned yet?
I've just used the find on the app to locate issues with matching content 👍

Would be great to **FIND IN PAGE** and see/ traverse results 🙂

Thank you!

**Comment by kamzil** [comment]
Thanks for creating this great app! That said, I find it very surprising that a core functionality as important as being able to search code is still missing after two years. It's just not "find in page". It's also not possible to use code search in general in the current Android app AFAICT. This together with adding zoom in/out and improving scroll behavior in code view would increase the usability of this app **a lot**.

**Comment by pomac303** [comment]
As it is, i try to avoid the app if i can just to be able to search... The lack of it is enough motivation to basically uninstall...

**Reply by tusharsnx** [reply]
Same. But, so far I've only blocked the redirecting when we browse to `github.com`. It was painful when you go to github.com for code search feature, but then it opens up the app, defeats the whole purpose right? 🙃

**Comment by CryptoHeni793** [comment]
G
